tests/pytest_tests/conftest.py

```diff
@@ -17,7 +17,7 @@
 from wandb import Api  # noqa: E402
 from wandb.sdk.interface.interface_queue import InterfaceQueue  # noqa: E402
 from wandb.sdk.lib import filesystem, runid  # noqa: E402
-from wandb.sdk.lib.git import GitRepo  # noqa: E402
+from wandb.sdk.lib.gitilb import GitRepo  # noqa: E402
```

Comment:
```suggestion
from wandb.sdk.lib.gitlib import GitRepo  # noqa: E402
```
While it is inline with the imported file/module name, I think it is typo (here and elsewhere)?

Reply:
oh thanks this is actually correct